### DISCUSSION TITLE
How to switch between Instances on Local Setup

### DISCUSSION THREAD

**Discussion by huzaifaali14** [zitadel/zitadel#4673] [Q&A] [answered]
I have created a new instance using rest endpoint and I can see the data in database as well. I am wondering is there a way to switch between the instances through console because I couldn't find any option? I have configured the Zitadel locally using docker compose. Also is there a way through which we can setup instances via console ?
<img width="1396" alt="Screenshot 2022-11-07 at 9 10 29 PM" src="https://user-images.githubusercontent.com/30002668/200358622-92b7fb9c-6a4e-4b72-917f-2fadbcec7253.png">

**Comment by eliobischof** [comment]
ZITADEL maps your requests Host header to the instance. So you need to access your new instance at http://\<your instances domain\>:8080/ui/console.

Managing instances is only supported using the system API, not via console.

**Comment by hifabienne** [comment, marked as answer]
Hi @huzaifaali14 

It is not possible to switch between the instances in the console or to setup them in the in the console.
The instances do not know anything from the other instances and also don't share any data. Due to this it is not possible to integrate this in the management UI. The console is only used to manage one instance.

Managing different instances and set them up ist the layer above we have implemented our ZITADEL Customer portal for this, which we are using for our SaaS offering.

**Comment by huzaifaali14** [comment]
Thank you guys for the prompt replies. It's clear now.
